src/Microsoft.ML.Transforms/TensorflowTransform.cs

```diff
@@ -0,0 +1,665 @@
+            }
+
+            if (tfType == TFDataType.Int32)
+            {
+                return Utils.MarshalInvoke(MakeGetter<int>, outInfo.ItemType.RawType, input, iinfo);
+            }
+
+            if (tfType == TFDataType.Int64)
+            {
+                return Utils.MarshalInvoke(MakeGetter<Int64>, outInfo.ItemType.RawType, input, iinfo);
+            }
+
+            if (tfType == TFDataType.Bool)
+            {
+                return Utils.MarshalInvoke(MakeGetter<bool>, outInfo.ItemType.RawType, input, iinfo);
+            }
+
+            if (tfType == TFDataType.String)
+            {
+                return Utils.MarshalInvoke(MakeGetter<string>, outInfo.ItemType.RawType, input, iinfo);
+            }
+
+            throw Host.ExceptNotSupp("Tensorflow type not supported");
+        }
+
+        private ValueGetter<VBuffer<T>> MakeGetter<T>(IRow input, int iinfo)
+        {
+            Host.AssertValue(input);
+            // Need to check type here.
+            //Host.Assert(Infos[iinfo].TypeSrc.IsText);
+
+            var info = _bindings.Infos[iinfo];
+            var tfInfo = _bindings.TfColInfo[iinfo];
+            var srcTensorGetters = GetTensorValueGetters(input, iinfo);
+
+            return (ref VBuffer<T> dst) =>
+                {
+                    var runner = _session.GetRunner();
+                    for (int i = 0; i < info.SrcIndices.Length; i++)
+                    {
+                        var inputName = tfInfo.InputColNames[i];
+                        var type = info.SrcTypes[i];
+                        runner.AddInput(inputName, srcTensorGetters[i].GetTensor());
+                    }
+
+                    var tensors = runner.Fetch(_bindings.OutputColNames[iinfo]).Run();
+
+                    Contracts.Assert(tensors.Length > 0);
+
+                    var output = FetchData<T>(tensors[0].Data, _bindings.OutputCols[iinfo].VectorSize);
+                    dst = new VBuffer<T>(output.Length, output);
+                };
+        }
+
+        protected override Func<int, bool> GetDependenciesCore(Func<int, bool> predicate)
+        {
+            return _bindings.GetDependencies(predicate);
+        }
+
+        protected override Delegate[] CreateGetters(IRow input, Func<int, bool> active, out Action disp)
+        {
+            Func<int, bool> activeInfos =
+                iinfo =>
+                {
+                    int col = _bindings.MapIinfoToCol(iinfo);
+                    return active(col);
+                };
+
+            var getters = new Delegate[_bindings.InfoCount];
+            disp = null;
+            using (var ch = Host.Start("CreateGetters"))
+            {
+                for (int iinfo = 0; iinfo < _bindings.InfoCount; iinfo++)
+                {
+                    if (!activeInfos(iinfo))
+                        continue;
+                    getters[iinfo] = MakeGetter(input, iinfo);
```

Comment:
>MakeGetter [](start = 37, length = 10)

Add an out parameter of a disposer to MakeGetter, this way it can be used here, and by the RowCursor.